commit 1f9882b38b046c7752ec3e1200b17b7edfc995cd
Author: Kacper Kornet <draenog@pld-linux.org>
Date:   Sat May 4 20:27:19 2013 +0100

    Don't use tagFunction during copying headers
    
    It fixes error when the changelog in subpackage was converted
    to encoding determined by environment.

diff --git a/rpmdb/header.c b/rpmdb/header.c
index 01056c3..1cec5cc 100644
--- a/rpmdb/header.c
+++ b/rpmdb/header.c
@@ -2213,7 +2213,7 @@ void headerCopyTags(Header headerFrom, Header headerTo, rpmTag * tagstocopy)
 	if (headerIsEntry(headerTo, *tagno))
 	    continue;
 	he->tag = *tagno;
-	if (!headerGet(headerFrom, he, 0))
+	if (!headerGet(headerFrom, he, HEADERGET_NOEXTENSION))
 	    continue;
 	xx = headerPut(headerTo, he, 0);
 	he->p.ptr = _free(he->p.ptr);
